db.go

```diff
@@ -0,0 +1,72 @@
+package auth
```

Comment:
putting this in the top level `auth` package for now. This will avoid circular deps and still make key package members accessible from the top level of this package.